src/main/kotlin/org/rust/ide/actions/mover/RsLineMover.kt

```diff
@@ -93,6 +91,12 @@ abstract class RsLineMover : LineMover() {
 
         fun isMovingOutOfMatchArmBlock(sibling: PsiElement, down: Boolean): Boolean =
             isMovingOutOfBraceBlock(sibling, down) && sibling.parent?.parent?.parent is RsMatchArm
+
+        fun isMovingOutOfBlockExprInsideArgList(sibling: PsiElement, down: Boolean): Boolean {
+            if (!isMovingOutOfBraceBlock(sibling, down)) return false
+            val blockExpr = sibling.parentOfType<RsBlockExpr>() ?: return false
+            return blockExpr.parentOfType<RsValueArgumentList>() != null
```

Comment:
```suggestion
            val blockExpr = sibling.ancestorStrict<RsBlockExpr>() ?: return false
            return blockExpr.ancestorStrict<RsValueArgumentList>() != null
```

Reply:
Just out of curiosity, why is this preferred? It seems to call the same function with the same arguments.

Reply:
Well, I guess there are several reasons for preferring this method
1. `PsiElement.parentOfType(withSelf: Boolean = true): T?` was introduced only in 201 platform. Before that, there was only `PsiElement.parentOfType(): T?`. So there was no good equivalent for `ancestorOrSelf` for a long time. Now I think it's better to be consistent and use the same methods for the same things
2. `ancestorStrict` and `ancestorOrSelf` have more explicit names and you don't have to remember what that boolean flag means

Reply:
To be honest, `strict` is also a bit confusing to me, but I agree that `ancestor` is a better name than `parent`. Thanks for the explanation :+1:

Btw CI seems to be stuck at this moment because of download issues.